pkg/client/userd/commands/intercept.go

```diff
@@ -191,6 +189,58 @@ func (c *interceptCommand) init(ctx context.Context) {
 	}
 }
 
+func (c *interceptCommand) validArgsFunc() ValidArgsFunction {
+	return func(ctx context.Context, cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		cs := GetConnectorServer(ctx)
+		if cs == nil {
+			dlog.Debug(ctx, "no connector found")
+			return nil, cobra.ShellCompDirectiveError
+		}
+
+		var (
+			namespaceArgsIdx int
+			namespaceArg     string
+			namespace        = "default"
+		)
+		for i := 0; i < len(args); i++ {
+			if strings.HasPrefix(args[i], "--namespace") {
```

Comment:
Should this check for the short form `-n` too?

Reply:
ah yeah good catch; thanks